src/site/content/en/blog/yahoo-japan-news-bfcache/index.md

```diff
@@ -0,0 +1,161 @@
+      <td>
+        <strong>Uplift % (desktop)</strong>
+      </td>
+    </thead>
+    <tbody>
+      <tr>
+        <td>bfcache hit rate</td>
+        <td>
+          <strong>+54.03 points</strong> (0.04% → 54.07%)
+        </td>
+        <td>+47.28 points (0.02% → 47.30%)</td>
+      </tr>
+      <tr>
+        <td>Page views</td>
+        <td>
+          <strong>+2.26%</strong>
+        </td>
+        <td>+0.65%</td>
+      </tr>
+      <tr>
+        <td>Ads revenue</td>
+        <td>
+          <strong>+9.0%</strong>
+        </td>
+        <td>+0.6%</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+When web pages become instant with bfcache, the users tend to stay on page longer, thus increasing the views on ads, leading to the increase of ads revenue.
+
+However, we think there is more to bfcache than just making the page instant.
+
+## bfcache enhances seamless user experience on the website
+
+bfcache is not just about making the pages instant. When the pages are instant, **the website navigation feels more seamless.**
+
+In Yahoo! JAPAN News, one of the major user journey is as follows:
+
+1. Go to the article list
+2. Click on one article to read
+3. Go back to the article list
+4. Click on another article to read
+
+Before bfcache, when users finished reading an article in (step 2), they had to wait for the article list page to load. This could be a friction factor for users who just want to go back to the article list to pick out another article to read.
+
+Another source of friction during backward navigation was the scroll position. In practice, the browser tries to restore the scroll position when a backward navigation happens. However, because of dynamically-added ads or other layout changes, the scroll position often gets incorrectly restored, which could cause users to lose their bearings or even leave the page. This is never an issue when a backward navigation is powered by bfcache: the scroll position is immediately and correctly restored.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/WlJn1GZnDrSLpEPPHWAn.png", alt="Two filmstrips of a backward navigation from an article to the article listing page. The top is a filmstrip of the process being handled with bfcache which takes 0.3 seconds, whereas the bottom is of the same process being handled without bfcache, which takes 3.3 seconds.", width="800", height="343" %}
+</figure>
+
+Now with bfcache, the friction in the user journey is gone—the users can seamlessly go back to the article list page and pick another article to read instantly.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pk7QmKzBA49ADdcJGE7q.gif", alt="An animated image showing the backward navigation flow from an article to the article listing page with and without bfcache. With bfcache, the backward navigation is not only faster, but the scroll position is accurately maintained. Without bfcache, these guarantees cannot be made.", width="800", height="450" %}
+</figure>
```

Comment:
I got VERY confused with this video, cause it's not showing what's in the text above.

```suggestion
Now with bfcache, the friction in the user journey is gone—the users can seamlessly go back to the article list page and pick another article to read instantly.

The same thing happens when users browse from one article directly to another and back:

<figure>
  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pk7QmKzBA49ADdcJGE7q.gif", alt="An animated image showing the backward navigation flow from an article to the article listing page with and without bfcache. With bfcache, the backward navigation is not only faster, but the scroll position is accurately maintained. Without bfcache, these guarantees cannot be made.", width="800", height="450" %}
</figure>
```